### DISCUSSION TITLE
How to set multi values for StyleClass property?

### DISCUSSION THREAD

**Discussion by msnoshain** [dotnet/maui#9472] [Q&A] [answered]
I want to set styles for Entry by CSS and it works fine when just using a single style.
`<Entry StyleClass="m20" />`

However when I try to set multi values like this:
`<Entry x:Name="ent">`
`<Entry.StyleClass>`
`<x:String>m20</x:String>`
`<x:String>char-limit32</x:String>`
`</Entry.StyleClass>`
`</Entry>`

it throws an excepiton:
![image](https://user-images.githubusercontent.com/85274323/184954148-c6378c4a-9bc6-4e79-a63f-d4956618a57a.png)

It seems that my approach to set multi values is wrong.
The type of StyleClass is IList. Does anyone know how to set this property correctly?

**Comment by sjordanGSS** [comment, marked as answer]
In Xamarin.Forms you set StyleClass with a comma-separated list of strings, I assume the same is true in Maui. This might work:
```
<Entry StyleClass="m20, char-limit32" />
```

**Reply by msnoshain** [reply]
Thank you. It works
